setup.py

```diff
@@ -38,7 +38,7 @@ def find_version(*file_paths):
     zip_safe=True,
     python_requires=">=3.6",
     install_requires=[
-        "cloudformation-cli>=0.1.10,<0.2",
+        "cloudformation-cli>=0.1.11,<0.2",
```

Comment:
would probably remove the <0.2 requirement as [modules feature bumped this](https://github.com/aws-cloudformation/cloudformation-cli/releases/tag/v0.2.1)

Reply:
hoped to limit the scope of this PR :)